lib/MyChartAPI.js

```diff
@@ -203,10 +207,53 @@ async function AddFutureWeeks(
     return results;
 }
 
+const CommonFilters = {
+    DaysOfWeek: {
+        0: true,
+        1: true,
+        2: true,
+        3: true,
+        4: true,
+        5: true,
+        6: true,
+    },
+    TimesOfDay: "both",
+
+    // these functions take a list of IDs and convert them to an object with the IDs as keys, and values all true.
+    Departments: (departments) =>
+        Object.fromEntries(departments.map((dept) => [dept, true])),
+    Providers: (providers) =>
+        Object.fromEntries(providers.map((dept) => [dept, true])),
```

Comment:
The arrow param should not be `dept` on L226 when it is handling things that aren't departments. Either make them both generic (and the same) or both specific (and different).